wallet-services/src/main/java/org/exoplatform/wallet/service/ApplicationRegistrySetupService.java

```diff
@@ -31,20 +31,9 @@
  * SpaceWallet portlet if the application registry was already populated
  */
 public class ApplicationRegistrySetupService implements StartableClusterAware {
-  private static final String        SPACE_WALLET_APP_ID       = "wallet/SpaceWallet";
-
-  private static final String        SPACE_WALLET_PORTLET_NAME = "SpaceWallet";
-
-  private static final List<String>  EVERYONE_PERMISSION_LIST  = Collections.singletonList("Everyone");
-
-  private static final String        OLD_WALLET_CATEGORY_NAME  = "EthereumWallet";
-
-  private static final String        WALLET_CATEGORY_NAME      = "Wallet";
 
   private ExoContainer               container;
 
-  private ApplicationRegistryService applicationRegistryService;
-
   private boolean                    isDone;
 
   public ApplicationRegistrySetupService(PortalContainer container) {
```

Comment:
this startable service should be deleted since it is like an upgrade plugin which creates and configures wallet category with its application in full date servers

Reply:
done !